MastodonSDK/Sources/MastodonCore/Service/API/CoreData/APIService+CoreData+InstanceV2.swift

```diff
@@ -0,0 +1,69 @@
+import os.log
+import Foundation
+import CoreData
+import CoreDataStack
+import MastodonSDK
+
+extension APIService.CoreData {
+    
+    static func createOrMergeInstanceV2(
```

Comment:
I prefer using the same pattern in the `./MastodonSDK/Sources/MastodonCore/Persistence/Persistence+MastodonUser.swift` file. The function is same but maybe better naming style.

Reply:
I'm not sure I understand this, you mean I should move the persistency code to something like `Persistence+Instance` ? Or is it something else you're talking about here?

Reply:
Ah I think now I got it, regarding the `PersistContext`, I'll update it to be similar.